src/main/java/com/iexec/worker/executor/TaskManagerService.java

```diff
@@ -40,6 +40,7 @@
 import com.iexec.worker.tee.TeeServicesManager;
 import com.iexec.worker.utils.LoggingUtils;
 import com.iexec.worker.utils.WorkflowException;
+import io.micrometer.core.instrument.util.StringUtils;
```

Comment:
Are you sure about that import?

Reply:
removed in 2fbe435, thanks for your awareness